doc/sphinx/source/env_variables.rst

```diff
@@ -161,6 +161,11 @@ pocl.
  Defaults to 1. If set to 0, pocl will not install the SIGFPE handler.
  See :ref:`sigfpe-handler`
 
+- **POCL_SYNC_CACHE**
```

Comment:
Is this really necessary? I mean 1) is the fdatasync() needed in some scenarios? 2) is there a reason not to call it by default? The cold cache case is slow anyways and it should affect that only. I'd like to reduce the number of envs we have instead of adding them unless really necessary.

Reply:
Well, it's not that necessary, but depending on the filesystem, a write+close+rename might result in an empty file in case of problems, and this lowers the chances of that. Good point about env vars though. I'll take another look at it.